cypress/support/fragments/finance/fiscalYears/fiscalYearDetails.js

```diff
@@ -0,0 +1,115 @@
+import {
+  Button,
+  HTML,
+  KeyValue,
+  Link,
+  MultiColumnListCell,
+  MultiColumnListRow,
+  PaneHeader,
+  Section,
+  including,
+} from '../../../../../interactors';
+
+const fiscalYearDetailsSection = Section({ id: 'pane-fiscal-year-details' });
+const fiscalYearDetailsHeader = PaneHeader({ id: 'paneHeaderpane-fiscal-year-details' });
+const actionsButton = fiscalYearDetailsHeader.find(Button('Actions'));
```

Comment:
Do we need this `actionsButton` here? It's not used in this file

Reply:
Done